### DISCUSSION TITLE
Links on the site to github are broken

### DISCUSSION THREAD

**Discussion by AlexeyDmitriev** [Homebrew/discussions#4704] [Everyday usage] [answered]
### Output of `brew config`

```shell
irrelevant
```

### Output of `brew doctor`

```shell
irrelevant
```

### Description of issue

example: https://formulae.brew.sh/cask/google-chrome
here link to github is https://github.com/Homebrew/homebrew-cask/blob/d50a9fa3835b8fbb62a3ca974cba95132a328229/Casks/google-chrome.rb while it should go to https://github.com/Homebrew/homebrew-cask/blob/d50a9fa3835b8fbb62a3ca974cba95132a328229/Casks/g/google-chrome.rb

since all of them were recently moved

**Comment by gromgit** [comment]
There's an ongoing effort to rearrange the core repo files into a deeper hierarchy; GitHub doesn't do well with displaying a single directory containing thousands of files. The `formulae.brew.sh` generator code seems to not have been updated to reflect this.

cc @Homebrew/formulae-brew-sh

**Reply by SMillerDev** [reply, marked as answer]
https://github.com/Homebrew/formulae.brew.sh/pull/949

**Reply by ZhongRuoyu** [reply]
Yes, this is fixed now: https://formulae.brew.sh/cask/google-chrome#default
